awscognito/cognito_users.py

```diff
@@ -0,0 +1,76 @@
+# pylint: disable=global-statement,redefined-outer-name
+""" Script used to create|delete AWS Cognito user """
+import argparse
+import sys
+
+
+def create_user(client, profile, user):
+    """ Creates a new user in the specified user pool """
+    response = client.admin_create_user(
```

Comment:
what would happen if the user already exists in the pool?

Reply:
UsernameExistsException will be given.

Reply:
And therefore, we can simply call create_user and catch the exception to ignore duplicated user.